src/graphics/window.rs

```diff
@@ -133,7 +139,17 @@ impl Window {
         &mut self,
         new_cursor: winit::window::CursorIcon,
     ) {
-        self.surface.window().set_cursor_icon(new_cursor);
+        if self.cursor_icon != new_cursor {
+            self.surface.window().set_cursor_icon(new_cursor);
+            self.cursor_icon = new_cursor;
+        }
+    }
```

Comment:
Then, we can make this take an `Option<winit::Window::CursorIcon>` and remove `set_cursor_visible`.

Reply:
Addressed in 66828daceee95b73aa10a45c2d279fe7600e3660.